ConversorMedidas/app/src/main/java/projetointegrado/com/conversormedidas/ConversorDistancia.java

```diff
@@ -20,11 +21,26 @@ private String formatarNumero(Double numero){
         BigDecimal value = new BigDecimal(numero);
         BigDecimal grande = new BigDecimal(1000000000); // Teto de valor para não notação científica
         BigDecimal pequeno = new BigDecimal(0.001); // Chão
+        BigDecimal zero = new BigDecimal(0);
+
+
+        if(numero == 0){
```

Comment:
Treatment when number is 0, it used to format the 0 as scientific notation but this if treats it the way it should, just zero